rubocop/lib/rubocop.yml

```diff
@@ -0,0 +1,335 @@
+require: rubocop-rspec
+
+AllCops:
+  Exclude:
+    - bin/**/*
+
+# LINT
+
+Lint/AmbiguousBlockAssociation:
+  # `scope :name, -> (param) { ... }` is a valid syntax
+  Enabled: false
+
+Lint/AmbiguousOperator:
+  # REVIEW
+  Enabled: true
+
+Lint/AssignmentInCondition:
+  # REVIEW
+  # It allows to use `if (foo = 123)` instead of `if foo = 123`
+  Enabled: true
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
+
+Lint/HandleExceptions:
+  Enabled: false
+
+Lint/LiteralInCondition:
+  # backtics? seriously?
+  Enabled: false
+
+Lint/RescueException:
+  Enabled: false
+
+Lint/ShadowingOuterLocalVariable:
+  Enabled: false
+
+# Metrics
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - spec/**/*
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/CyclomaticComplexity:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 100
+  AllowHeredoc: false
+  AllowURI: false
+
+Metrics/MethodLength:
+  Max: 20
+
+Metrics/PerceivedComplexity:
+  Enabled: false
+
+# Performance
+
+Performance/Casecmp:
+  Enabled: false
+
+# RSpec
+
+RSpec/AnyInstance:
+  Enabled: false
+
+RSpec/ExampleLength:
+  Max: 20
+
+RSpec/DescribeClass:
+  Enabled: false
+
+RSpec/DescribedClass:
+  Enabled: false
+
+RSpec/DescribeMethod:
+  Enabled: false
+
+RSpec/LetSetup:
+  Enabled: false
+  
+RSpec/MessageSpies:
+  EnforcedStyle: receive
+
+RSpec/MultipleDescribes:
+  Enabled: false
+
+RSpec/MultipleExpectations:
+  Enabled: false
+
+RSpec/NestedGroups:
+  Enabled: false
+
+RSpec/VerifiedDoubles:
+  Enabled: false
+
+# Style
+
+Style/AccessorMethodName:
+  # This cop assumes every method is accessor
+  Enabled: false
+
+Style/AlignParameters:
+  # REVIEW
+  Enabled: true
+
+Style/AndOr:
+  # `do_something and return`
+  Enabled: false
+
+Style/CommentAnnotation:
+  # Also, i think that better to use yard's `@todo` and other in ruby code
+  Enabled: false
+
+Style/ConditionalAssignment:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Style/DoubleNegation:
+  # Always return booleans in predicates
+  Enabled: false
+
+Style/EmptyCaseCondition:
+  # REVIEW: let's enable it
```

Comment:
I prefer blank `case` because it’s more clear that all conditions are on the same level (`if` and `elsif` have different length).